config/v12/tca-120.php

```diff
@@ -10,4 +10,5 @@
     $rectorConfig->import(__DIR__ . '/../config.php');
     $rectorConfig->rule(MigrateNullFlagRector::class);
     $rectorConfig->rule(MigrateRequiredFlagRector::class);
+    $rectorConfig->rule(\Ssch\TYPO3Rector\Rector\v12\v0\tca\RemoveTCAInterfaceAlwaysDescriptionRector::class);
```

Comment:
please try to remember to import that namespace manually.

Reply:
ah yes